src/classes/BDI_DataImportFLSService.cls

```diff
@@ -133,14 +137,18 @@ public class BDI_DataImportFLSService {
     }
 
     private Boolean isValid(SObjectField field) {
+        Set<SObjectField> fieldSet = new Set<SObjectField>{
+                field
+        };
         if ((accessLevels.contains(AccessLevel.VALIDATE_CREATE)
-                && !field.getDescribe().isCreateable())
+                && !permissionsService.canCreate(fieldSet))
```

Comment:
I honestly could go either way on this, but for the purpose of discourse - is it worth adding methods to UTIL_Permissions to test single SObjectField? Like `canCreate(SObjectField field)`?

Reply:
Yeah, it would look cleaner from the calling location in this instance, but you can obviously accomplish what is needed by passing in a collection, even of size 1, like here.  I could go either way also.